### DISCUSSION TITLE
Self hosted with openlitespeed?

### DISCUSSION THREAD

**Discussion by lightson** [plausible/analytics#1984] [Self-Hosted Support] [answered]
Are there any guides/anyone that's been successful getting self hosted Plausible running on openlitespeed?

It would be great if there were install methods outside of docker.

**Comment by lightson** [comment, marked as answer]
Sorted it out. If forgotten how to properly configure a reverse proxy, so that was easily resolved.

**Reply by zenminimalist** [reply]
Care to share? I'm trying to get it to run but just can't with my limited knowledge : (
